src/unix/linux/process.rs

```diff
@@ -660,6 +660,7 @@ fn get_all_pid_entries(
     parent_pid: Option<Pid>,
     entry: DirEntry,
     data: &mut Vec<ProcAndTasks>,
+    refresh_kind: ProcessRefreshKind,
```

Comment:
Instead of passing the whole `ProcessRefreshKind`, why not just passing the bool returned from `refresh_kind.thread()`?

Reply:
done